src/components/organisms/ConversationWindow.vue

```diff
@@ -33,8 +33,17 @@
         aria-live="polite"
       >
         <li>
-          <p class="text-center font-heading text-sm font-light text-gray-dark">
-            {{ $t("messageTime") }}
+          <p
+            class="
+              text-center
+              uppercase
+              font-heading
+              text-sm
+              font-light
+              text-gray-dark
+            "
+          >
+            {{ messageTimestamp || $t("messageTime") }}
```

Comment:
I would remove the `$t("messageTime")` including the one in the locale files as we do not want to show a random hardcoded date, even if there are no messages. Instead we could return a new Date or something from the function you created. (See next comment)